nano/templates/autolathe.tmpl

```diff
@@ -0,0 +1,332 @@
+<!-- 
+Title: Autolathe UI 
+Used In File(s): \code\game\machinery\autolathe.dm
+-->
+
+<style type='text/css'>
+	#costsmall
+	{
+		width:250px;
+		float:left;
+	}
+	#misc
+	{
+		width:350px;
+		float:left;
+	}
+</style>
+
+<div style="float: right; width: 75%;">
+	<div class="itemLabel">
+		Queue
+	</div>
+	<div class="itemContentWide" style="width: 95%;">
+		<div class="statusDisplay">
+			<div class="line" style="overflow-y: auto; overflow-x: hidden; scrollbar-base-color: #336699; scrollbar-arrow-color: #ffffff; height: 165px">
+				{{for data.queue}}
+				<div class="statusValue">
+					{{:helper.link(value.name, 'gear', null, null, null, 'misc')}}
+					{{:helper.link("Remove", 'circle-arrow-s', value.commands, null, 'fixedLeftWide')}}
+				</div>
+				{{empty}}
+					<i>No parts in memory</i>
+				{{/for}}
+			</div>
+			<br>
+			<div class="line">
+				{{:helper.link('Process Queue', 'circle-arrow-s', {'process_queue':null}, null, null, 'misc')}}
+				{{:helper.link('Clear Queue', 'circle-arrow-s', {'clear_queue':null}, null, 'fixedLeft')}}
+			</div>
+		</div>
+	</div>
+</div>
+<div style="float: left; width: 25%;">
+	<div class="itemLabel">
+		Categories
+	</div>
+	<div class="line">
+		{{:helper.link('Tools',		 	'gear',{'screen':11},(data.screen==11)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Containers',	'gear',{'screen':12},(data.screen==12)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Assemblies',	 'gear',{'screen':21},(data.screen==21)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Stock Parts',	 'gear',{'screen':22},(data.screen==22)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Medical',		 'gear',{'screen':31},(data.screen==31)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Ammunition',	 'gear',{'screen':32},(data.screen==32)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Misc. Tools',	 'gear',{'screen':41},(data.screen==41)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	<div class="line">
+		{{:helper.link('Misc. Items',	 'gear',{'screen':42},(data.screen==42)?'linkOn':'', 'fixedLeftWide')}}
+	</div>
+	{{if data.hacked == 01}}
```

Comment:
If you want to do it like this, you have to add a ref check to Topic and change the update_hacked() for the autolathe because it already adds Hidden Items to Items when it gets hacked.